src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.tsx

```diff
@@ -16,7 +17,13 @@ const NearestCitiesWeatherWidget = ({ cityName, coordinates }: Props) => {
     }
 
     if (data) {
-      return data.map((city) => <CityWeatherShortcut key={city.id} cityName={city.tags.name} />);
+      return data.map((city) => (
+        <CityWeatherShortcut
+          key={city.id}
+          cityName={city.tags.name}
+          button={<FavoritesButton cityName={cityName} size={20} />}
```

Comment:
**todo**  I think there should be `city.tags.name` inside `cityName` props.
I did manual testing and I saw that if you add to favorites some city, all hearts are changing to red.

Reply:
Right, my mistake